auto_submit/lib/requests/cronjob_handler.dart

```diff
@@ -0,0 +1,185 @@
+// Copyright 2022 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:convert';
+
+import 'package:auto_submit/service/config.dart';
+import 'package:github/github.dart';
+import 'package:shelf/shelf.dart';
+
+import '../service/github_service.dart';
+import '../service/log.dart';
+import '../server/request_handler.dart';
+
+/// Handler for processing GitHub webhooks.
+///
+/// On events where an 'autosubmit' label was added to a pull request,
```

Comment:
Hi Casey, if I understand correctly, I think the webhook handler will check and push all pull requests with 'autosubmit' label into the pubsub queue. The cronjob api will keep pulling the pubsub queue. It will help check whether the PR in queue should be merged or not.